DCode.Data/DbContexts/ApprovedApplicantDbContext.cs

```diff
@@ -0,0 +1,20 @@
+﻿using DCode.Common;
+using System.Data.Entity;
+using System.Data.Entity.Infrastructure;
+
+namespace DCode.Data.DbContexts
+{
+   public class ApprovedApplicantDbContext: DbContext
+    {
+        public ApprovedApplicantDbContext()
+            : base(Constants.DCodeEntities)
+        {
+            this.Configuration.LazyLoadingEnabled = false;
+        }
+        protected override void OnModelCreating(DbModelBuilder modelBuilder)
+        {
+            throw new UnintentionalCodeFirstException();
```

Comment:
Hey @priyankalekkala ,
Please check this I am not confident if we should throw unhandled exception here.